projects/vdk-control-cli/cicd/build.sh

```diff
@@ -30,6 +30,7 @@ pip install -e .
 
 echo "Run unit tests and generate coverage report"
 pip install pytest-cov
+pip install pytest==6.2.4 # TODO: there is some regression with the newest version, remove this when possible
```

Comment:
What is the regression? Can you link an issue/ticket to pytest  
Do we need to open an internal github issue to address this?

Reply:
Please add this information in the comment or at least git description. We should be clear to future developer why we decided to make something if it's not clear from the code itself . 

Write in the ticket https://github.com/pytest-dev/pytest/issues/8539 that you are still seeing the issue in python v 3.10.xyz and in pytest vx.y ; Include snippet of logs and some details to help them root-cause.

Reply:
I was accidentally using an older version of pytest due to it being pinned in requirements.txt. Removed the pin and the extra pytest install, hopefully it should work. Any idea why it was pinned originally?

Reply:
No idea. That's why it's important to put comments. It was copied over from the previous repo, so git history is gone to see details in the commit message (if there were any). Likely for no specific reason (copy pasted from somewhere else :) )